packages/terra-section-header/src/SectionHeader.jsx

```diff
@@ -153,17 +153,28 @@ class SectionHeader extends React.Component {
     /* eslint-disable jsx-a11y/click-events-have-key-events */
     /* eslint-disable jsx-a11y/no-static-element-interactions */
 
-    const buttonAttributes = (onClick) ? { role: 'button', 'aria-expanded': isOpen, 'aria-label': headerText } : undefined;
+    // Function to use Arrange component
+    const ArrangeComponent = () => (
+      <Arrange
+        fitStart={onClick && accordionIcon}
+        fill={<span aria-hidden={(onClick !== undefined)} className={cx('title', 'title-block')}>{headerText}</span>}
+        className={cx('title-arrange')}
+      />
+    );
+
+    const buttonAttributes = (onClick) ? { 'aria-expanded': isOpen, 'aria-label': headerText } : undefined;
```

Comment:
Are these still needed? Should `aria-expanded': isOpen` be set on the `<button>` and `'aria-label': headerText` be set on the `<span>`?

Reply:
I realized I misread this. I don't think we need the ternary anymore since we can set `{ 'aria-expanded': isOpen, 'aria-label': headerText }` directly on the `<button>`.